sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Release History
+## Release History
```

Comment:
Quick question, usually this is generated by script? why the format is off?

Reply:
Yeah, the script is generating this in the order of H1s (H1 header) as the title, then H2 headers for release versions. However, since we use this file in our Microsoft docs here - https://github.com/MicrosoftDocs/azure-docs-pr/blob/master/articles/cosmos-db/sql/sql-api-sdk-java-v4.md
So this H1 header collides with existing H1 headers on official Microsoft Docs.  And it breaks it. 

There is an active issue to fix it which is being tracked by @weshaggard

Reply:
Yes we have https://github.com/Azure/azure-sdk-for-java/issues/23908 which will hopefully address this issue. cc @chidozieononiwu